src/app/zap-templates/zcl/data-model/chip/joint-fabric-datastore-cluster.xml

```diff
@@ -0,0 +1,370 @@
+      <access op="read" privilege="administer"/>
+    </attribute>
+    <attribute side="server" code="0x000D" name="NodeEndpointList" define="NODE_ENDPOINT_LIST" type="array" entryType="DatastoreEndpointEntryStruct">
+      <access op="read" privilege="administer"/>
+    </attribute>
+
+    <command source="client" code="0x00" name="AddKeySet" isFabricScoped="true">
+      <description>This command SHALL be used to add a KeySet to the Joint Fabric Datastore Cluster.</description>
+      <arg id="0" name="GroupKeySet" type="DatastoreGroupKeySetStruct"/>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="client" code="0x01" name="UpdateKeySet" isFabricScoped="true">
+      <description>This command SHALL be used to update a KeySet in the Joint Fabric Datastore Cluster.</description>
+      <arg id="0" name="GroupKeySet" type="DatastoreGroupKeySetStruct"/>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="client" code="0x02" name="RemoveKeySet" isFabricScoped="true">
+      <description>This command SHALL be used to remove a KeySet from the Joint Fabric Datastore Cluster.</description>
+      <arg id="0" name="GroupKeySetID" type="int16u"/>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="client" code="0x03" name="AddGroup" isFabricScoped="true">
+      <description>This command SHALL be used to add a group to the Joint Fabric Datastore Cluster.</description>
+      <arg id="0" name="GroupID" type="group_id"/>
+      <arg id="1" name="FriendlyName" type="char_string" length="32"/>
+      <arg id="2" name="GroupKeySetID" type="int16u" isNullable="true" min="1" max="65534"/>
+      <arg id="3" name="GroupCAT" type="int16u" isNullable="true" min="1" max="65534"/>
+      <arg id="4" name="GroupCATVersion" type="int16u" isNullable="true" min="1" max="65534"/>
+      <arg id="5" name="GroupPermission" type="DatastoreAccessControlEntryPrivilegeEnum" min="0x00" max="0x05"/>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="client" code="0x04" name="UpdateGroup" isFabricScoped="true">
+      <description>This command SHALL be used to update a group in the Joint Fabric Datastore Cluster.</description>
+      <arg id="0" name="GroupID" type="group_id"/>
+      <arg id="1" name="FriendlyName" type="char_string" length="32" isNullable="true"/>
+      <arg id="2" name="GroupKeySetID" type="int16u" isNullable="true" min="1" max="65534"/>
+      <arg id="3" name="GroupCAT" type="int16u" isNullable="true" min="1" max="65534"/>
+      <arg id="4" name="GroupCATVersion" type="int16u" isNullable="true" min="1" max="65534"/>
+      <arg id="5" name="GroupPermission" type="DatastoreAccessControlEntryPrivilegeEnum" min="0x00" max="0x05"/>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="client" code="0x05" name="RemoveGroup" isFabricScoped="true">
```

Comment:
Spec issue: This also has broken text:

```
Attempts to remove a group with GroupCAT value set to <ref_JFAdministratorCAT,Administrator CAT> or <ref_JFAnchorCAT,Anchor CAT> SHALL fail.
```

Again: what does "SHALL fail" mean?

Reply:
Tracked by https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/11620

Reply:
The spec change will be tracked using [issue #11620](https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/11620). Once the spec update is complete, a follow-up SDK PR can be submitted to align it with the updated specification.